SU-SWS/stanford_profile_helper
# D8CORE-4178: Fixing the h2 that wasn't wrapping

# READY FOR REVIEW

# Summary
- Removing the display: inline-block to allow for titles wrapping. 

# Needed By (Date)
- 5/15

# Urgency
- Med

# Steps to Test

1. Checkout this branch.
2. Clear caches.
3. Create some Basic Page Plus pages. Some long and short titles. 
4. Add a list of card and default display to a page with the Research Project filter.
5. Notice that the long titles wrap.[ Example page in the wild.](https://ewinggroup2.sites.stanford.edu/research)
6. Verify nothing else breaks.
7. @imonroe, I couldn't figure out why this was there. Were you fixing a case I couldn't find? Let me know?

# Affected Projects or Products
- Basic Page Plue

# Associated Issues and/or People
- D8CORE-4178

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
